randommm.py

```diff
@@ -0,0 +1,4 @@
+from random import randint
```

Comment:
We can probably delete this file for now.

Reply:
Should I delete it here in github?

Reply:
You either need to:
- delete it here and then fetch+rebase locally
- delete it locally and push the change

They are totally equivalent, so do whichever you prefer.

I always get things how I want them locally and then push. I much prefer the terminal to any UI abstractions and I like to make sure everything is perfect before I make it publicly visible.

Reply:
I think I agree with you on that.

Reply:
Feel free to click the "Resolve conversation" button for this thread.